statsmodels/tools/tools.py

```diff
@@ -18,17 +18,6 @@ def asstr2(s):
         return str(s)
 
 
-def _make_dictnames(tmp_arr, offset=0):
-    """
-    Helper function to create a dictionary mapping a column number
-    to the name in tmp_arr.
-    """
-    col_map = {}
-    for i, col_name in enumerate(tmp_arr):
-        col_map[i + offset] = col_name
-    return col_map
-
```

Comment:
This function was removed because I noticed that this function was not being used anywhere, and therefore my code editor's linter (Ruff) raised a warning about this.